### DISCUSSION TITLE
Moving items up and down

### DISCUSSION THREAD

**Discussion by Fymyte** [nvim-neorg/neorg#372] [Q&A] [answered]
Hi, first thanks for this piece of software, just 👌
I have a keybinding in my config which allows me to move lines or block of lines around, but it seems to be overritten
by neorg and gives me an error:
```Unable to trigger keybind core.norg.manoeuvre.item_down - the keybind does not exist```
I use "Alt+j" for moving lines down

I never requested the manoeuvre module explicitly

Am I missing a configuration somewhere ?

**Comment by mrossinek** [comment, marked as answer]
I'm not 100% certain I'm understanding everything but let me try to answer.

I assume you mean you have non-Neorg related keybinding mapped to `Alt+j` which does not work within Neorg.
Within Neorg it instead raises the error above. Is that correct?

If so, then this is likely because the default keybinds include a map to this keybinding, here: https://github.com/nvim-neorg/neorg/blob/3533008bfd4b276d1a8ff8db017fa014fda64066/lua/neorg/modules/core/keybinds/keybinds.lua#L35

As I see things there are two things:
1. You can disable the default Neorg keybind. See https://github.com/nvim-neorg/neorg/wiki/User-Keybinds#setting-up-a-keybind-hook and use the `unmap` function
2. Neorg should respect existing maps instead of forcing itself onto the user. @vhyrro shouldn't this already be done?

**Comment by Fymyte** [comment]
Yep this is exactly my issue.
I added

```lua
neorg.setup {
  -- Tell Neorg what modules to load
  load = {
    ...
    ["core.keybinds"] = {
      config = {
        hook = function (keybinds)
          keybinds.unmap('norg', 'n', '<M-j>')
          keybinds.unmap('norg', 'n', '<M-k>')
        end,
      },
    },
    ...
  },
} 
```
In my config and it works properly now 👍

- Neovim version:
```
NVIM v0.7.0-dev+1093-g9ef569613
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
````
- Neorg is up-to-date with main (commit d6d6446)
